go.mod

```diff
@@ -9,20 +9,21 @@ require (
 	github.com/fatih/structs v1.1.0
 	github.com/go-clix/cli v0.1.1
 	github.com/gobwas/glob v0.2.3
-	github.com/google/go-cmp v0.5.2-0.20200818193711-d2fcc899bdc2
+	github.com/google/go-cmp v0.5.4
 	github.com/google/go-jsonnet v0.17.0
 	github.com/huandu/xstrings v1.3.2 // indirect
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/karrick/godirwalk v1.15.5
 	github.com/pkg/errors v0.9.1
 	github.com/posener/complete v1.2.3
+	github.com/prometheus/prometheus v1.8.2-0.20210215121130-6f488061dfb4
```

Comment:
v1.8.2 appears to be a fairly old version of Prometheus. Is this on purpose?

Reply:
That's a side effect of how Go Modules deal with version tags. It's actually a current version used here (from 2021-02-15, see timestamp).